bower.json

```diff
@@ -25,5 +25,8 @@
     "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
     "jquery": "~2.1.1",
     "ember-data": "~1.0.0-beta.10"
+  },
+  "resolutions": {
+    "ember-data": "canary"
```

Comment:
You mean you can't just cater to my exact setup?? haha :wink: 

Let me know if theres anything I can do to help, thanks for your guys' hard work!